cypress/e2e/regression/sidebar.cy.js

```diff
@@ -0,0 +1,51 @@
+import * as constants from '../../support/constants'
+import * as main from '../pages/main.page'
+import * as sideBar from '../pages/sidebar.pages'
+import * as navigation from '../pages/navigation.page'
+
+describe('Sidebar tests', () => {
+  beforeEach(() => {
+    cy.visit(constants.homeUrl + constants.SEPOLIA_TEST_SAFE_13)
+    cy.clearLocalStorage()
+    main.acceptCookies()
+  })
+
+  it('Verify Current network is displayed at the top', () => {
+    sideBar.verifyNetworkIsDisplayed(constants.networks.sepolia)
+  })
+
+  it('Verify current safe details', () => {
+    sideBar.verifySafeHeaderDetails(sideBar.testSafeHeaderDetails)
+  })
+
+  it('Verify QR button opens the QR code modal', () => {
+    sideBar.clickOnQRCodeBtn()
+    sideBar.verifyQRModalDisplayed()
+  })
+
+  it('Verify Copy button copies the address', () => {
+    sideBar.verifyCopyAddressBtn(constants.SEPOLIA_TEST_SAFE_13.substring(4))
+  })
+
+  it('Verify Open blockexplorer button contain etherscan link', () => {
+    sideBar.verifyEtherscanLinkExists()
+  })
+
+  it('Verify New transaction button enabled for owners', () => {
+    sideBar.verifyNewTxBtnStatus(constants.enabledStates.enabled)
+  })
+
+  it('Verify New Transaction button disabled for non-owners', () => {
```

Comment:
The button can be enabled for non-owner if the non-owner is a beneficiary.
The case should be added either here or in the spending limits non -owner tests

Reply:
This is done.